src/login/components/LoginFormLedger/LoginFormLedger.js

```diff
@@ -66,36 +68,20 @@ export default class LoginFormLedger extends React.Component {
     );
   }
 
-  renderDeviceInfo = () => {
-    const { deviceInfo } = this.props;
+  renderStatus = () => {
+    const { publicKey, deviceError, deviceInfo, progress } = this.props;
 
-    if (deviceInfo) {
-      return <p>Found USB {deviceInfo.manufacturer} {deviceInfo.product}</p>;
+    if (deviceError !== 'No USB device found.') {
```

Comment:
By wrapping the error check as the outer condition, this means that if an error happens once followed by success, I believe we'll always end up in the `else` condition where we show the "Searching for USB devices" messaging.